config/virtuoso/virtuoso.ini

```diff
@@ -0,0 +1,262 @@
+ErrorLogLevel			= 7
+FileExtend			= 200
+MaxCheckpointRemap		= 2000
+Striping			= 0
+TempStorage			= TempDatabase
+
+
+[TempDatabase]
+DatabaseFile			= /usr/local/virtuoso-opensource/var/lib/virtuoso/db/virtuoso-temp.db
+TransactionFile			= /usr/local/virtuoso-opensource/var/lib/virtuoso/db/virtuoso-temp.trx
+MaxCheckpointRemap		= 2000
+Striping			= 0
+
+
+;
+;  Server parameters
+;
+[Parameters]
+ServerPort			= 1111
+LiteMode			= 0
+DisableUnixSocket		= 1
+DisableTcpSocket		= 0
+;SSLServerPort			= 2111
+;SSLCertificate			= cert.pem
+;SSLPrivateKey			= pk.pem
+;X509ClientVerify		= 0
+;X509ClientVerifyDepth		= 0
+;X509ClientVerifyCAFile		= ca.pem
+MaxClientConnections		= 10
+CheckpointInterval		= 60
+O_DIRECT			= 0
+CaseMode			= 2
+MaxStaticCursorRows		= 5000
+CheckpointAuditTrail		= 0
+AllowOSCalls			= 0
+SchedulerInterval		= 10
+DirsAllowed			= ., /usr/local/virtuoso-opensource/share/virtuoso/vad
+ThreadCleanupInterval		= 10
+ThreadThreshold			= 10
+ResourcesCleanupInterval	= 10
+FreeTextBatchSize		= 100000
+SingleCPU			= 0
+VADInstallDir			= /usr/local/virtuoso-opensource/share/virtuoso/vad/
+PrefixResultNames               = 0
+RdfFreeTextRulesSize		= 100
+IndexTreeMaps			= 256
+MaxMemPoolSize                  = 200000000
+PrefixResultNames               = 0
+MacSpotlight                    = 0
+IndexTreeMaps                   = 64
+MaxQueryMem 		 	= 2G		; memory allocated to query processor
+VectorSize 		 	= 1000		; initial parallel query vector (array of query operations) size
+MaxVectorSize 		 	= 1000000	; query vector size threshold.
+AdjustVectorSize 	 	= 0
+ThreadsPerQuery 	 	= 4
+AsyncQueueMaxThreads 	 	= 10
+;;
+;; When running with large data sets, one should configure the Virtuoso
+;; process to use between 2/3 to 3/5 of free system memory and to stripe
+;; storage on all available disks.
+;;
+;; Uncomment next two lines if there is 2 GB system memory free
+;NumberOfBuffers          = 170000
+;MaxDirtyBuffers          = 130000
+;; Uncomment next two lines if there is 4 GB system memory free
+;NumberOfBuffers          = 340000
+; MaxDirtyBuffers          = 250000
+;; Uncomment next two lines if there is 8 GB system memory free
+;NumberOfBuffers          = 680000
+;MaxDirtyBuffers          = 500000
+;; Uncomment next two lines if there is 16 GB system memory free
+;NumberOfBuffers          = 1360000
+;MaxDirtyBuffers          = 1000000
+;; Uncomment next two lines if there is 32 GB system memory free
+;NumberOfBuffers          = 2720000
+;MaxDirtyBuffers          = 2000000
+;; Uncomment next two lines if there is 48 GB system memory free
+;NumberOfBuffers          = 4000000
+;MaxDirtyBuffers          = 3000000
+;; Uncomment next two lines if there is 64 GB system memory free
+;NumberOfBuffers          = 5450000
+;MaxDirtyBuffers          = 4000000
+;;
+;; Note the default settings will take very little memory
+;; but will not result in very good performance
+;;
+NumberOfBuffers          = 10000
```

Comment:
this is super low, we should probably use something a little higher already, like the 4GB version or so would alsoo increase vector size, max vector size and maxquerymem.
ofc these settings will change (and likely increase further) during the project